packages/cli/config/index.ts

```diff
@@ -22,6 +22,26 @@ const config = convict({
 			default: '',
 			env: 'DB_TABLE_PREFIX',
 		},
+		logging: {
+			enabled: {
+				doc: 'Typeorm logging enabled flag.',
+				format: 'Boolean',
+				default: false,
+				env: 'DB_LOGGING_ENABLED',
+			},
+			options: {
+				doc: 'Comma separated logging options, default is "error".',
+				format: String,
+				default: 'error',
+				env: 'DB_LOGGING_OPTIONS',
+			},
+			maxQueryExecutionTime: {
+				doc: 'Maximum number of milliseconds query should be executed before logger log a warning. Set 0 to disable',
```

Comment:
By disabled we mean all queries will be logged or no queries will be logged? I am a bit confused.

Reply:
if a query takes longer than specified, there will be warning logged. if set to 0, no logs would be generated. This is separate from enabling log btw, regardless of logs being enabled or not and regardless of the log level, if a query takes longer than specified, warn log is generated.

Reply:
https://typeorm.io/#/logging/enabling-logging